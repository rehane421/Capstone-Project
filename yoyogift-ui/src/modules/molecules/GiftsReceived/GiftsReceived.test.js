import React from "react";
import { shallow } from "../../../enzyme";
import { shallowToJson } from "enzyme-to-json";
import { GiftsReceived } from "./GiftsReceived";

describe("GiftsReceived", () => {
  let wrapper, props;

  props = {
    data: [],
    classes: {
      root: "root"
    }
  };
  it("should render GiftReceived", () => {
    wrapper = shallow(<GiftsReceived {...props} />);
  });
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
