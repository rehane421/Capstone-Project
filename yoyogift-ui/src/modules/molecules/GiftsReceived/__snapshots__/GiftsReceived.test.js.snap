// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GiftsReceived should render GiftReceived 1`] = `
<WithStyles(ForwardRef(Paper))
  className="root"
>
  <WithStyles(ForwardRef(Table))>
    <WithStyles(ForwardRef(TableHead))>
      <WithStyles(ForwardRef(TableRow))>
        <WithStyles(ForwardRef(TableCell))>
          CARD NAME
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          POINTS
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          RECEIVED FROM
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          CARD DESC
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          ISSUE DATE
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          EXPIRY DATE
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          REDEEM
        </WithStyles(ForwardRef(TableCell))>
      </WithStyles(ForwardRef(TableRow))>
    </WithStyles(ForwardRef(TableHead))>
    <WithStyles(ForwardRef(TableBody))>
      <WithStyles(ForwardRef(TableRow))
        key="1"
      >
        <WithStyles(ForwardRef(TableCell))
          component="th"
          scope="row"
        >
          Flipkart
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell)) />
        <WithStyles(ForwardRef(TableCell)) />
        <WithStyles(ForwardRef(TableCell)) />
        <WithStyles(ForwardRef(TableCell))>
          NaN-NaN-NaN
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          NaN-NaN-NaN
        </WithStyles(ForwardRef(TableCell))>
        <WithStyles(ForwardRef(TableCell))>
          <WithStyles(ForwardRef(Button))
            color="primary"
            variant="contained"
          >
            Redeem
          </WithStyles(ForwardRef(Button))>
        </WithStyles(ForwardRef(TableCell))>
      </WithStyles(ForwardRef(TableRow))>
    </WithStyles(ForwardRef(TableBody))>
  </WithStyles(ForwardRef(Table))>
</WithStyles(ForwardRef(Paper))>
`;
